packages/sdk/src/users/interfaces/Position.ts

```diff
@@ -18,6 +18,7 @@ export interface Position {
   positionId: PositionId
   debtAmount: TokenAmount
   collateralAmount: TokenAmount
-  riskRatio: RiskRatio
+  // riskRatio: RiskRatio // Damian: RiskRatio should be derived from debtAmount and collateralAmount, Position.utils.riskRatio(position)
+  // in order to calculate it, it should get prices from the pool.
```

Comment:
Remember that all price fetching will happen in the server. And also that we can pre-calculate things for frontend in the backend. For example `RiskRatio`, even if it can be calculated, we could provide it for FE so they can just show it, for example.